### DISCUSSION TITLE
Global CSS Opacity fields get set to 1% on next build.

### DISCUSSION THREAD

**Discussion by aaronhawkey48** [vercel/next.js#13069] [Help] [answered]
I have global styles for my project set. When I run `next` on my project, my styles are fine.

When I build the application using `next build` and run using `next start` and try to serve it, all my opacity fields get set to `opacity: 1%;`  

I uncovered this in my initial staging deploy of my application on firebase: 
https://nh-companion.web.app/

https://nh-companion.web.app/sign-in

https://nh-companion.web.app/feed is where you can see the error on the .card global style. 
Auth credentials to log in and see:

`
email: asdf@asdf.com
password: asdfasdf
`

Been stuck on this for quite a while. I just upgraded to see if this will fix the issue, but no success. On 9.4.1.

Screen cap of the issue in action.
https://www.loom.com/share/4cd941c50fd24ef8a706c972f376389c

styles.css (Global)
```
body {
    margin: 0 auto;
    --primary-main-color: #148AA8;
    --primary-light-color: #1F8FAC;
    --primary-background-color: #EEF7FF;
    --primary-border-radius: 10px;
}
p {
    margin: 0;
}
html {
    font-family: Arial, Helvetica, sans-serif;
    background-color: #EEF7FF;
}

.card {
    background-color: #ffffff;
    opacity: 95%;
}

textarea {
    font-family: Arial, Helvetica, sans-serif;
}

.noselect {
    -webkit-touch-callout: none; /* iOS Safari */
      -webkit-user-select: none; /* Safari */
       -khtml-user-select: none; /* Konqueror HTML */
         -moz-user-select: none; /* Old versions of Firefox */
          -ms-user-select: none; /* Internet Explorer/Edge */
              user-select: none; /* Non-prefixed version, currently
                                    supported by Chrome, Edge, Opera and Firefox */
}
  
.MuiFormLabel-root{
    color: var(--primary-main-color);
}
```

Post.tsx
```
import React from 'react';
import styles from './Post.module.css'
import PostHeader from './PostHeader/PostHeader';
import TextBody from './TextBody/TextBody';
import EventBody from './EventBody/EventBody';
import { PostHeaderProps } from './PostHeader/PostHeader';
import { TextBodyProps } from './TextBody/TextBody';
import { EventBodyProps } from './EventBody/EventBody';
import cn from 'classnames';


interface PostProps {
    type: string,
    postHeader: PostHeaderProps,
    textBody?: TextBodyProps,
    eventBody?: EventBodyProps,
};

const Post  = (props: PostProps) => {

    let postBody;

    switch(props.type) {
        case 'text':
            postBody = (
                <TextBody {...props.textBody} />
            );
            break;
        case 'event':
            postBody = (
                <EventBody 
                    {...props.eventBody}
                />
            );
            break;
        default:
            break;
    }

    return (
        <div className={styles.postContainer}>
            <div className={cn(styles.post, 'card')}>
                <PostHeader {...props.postHeader}/>
                {postBody}
            </div>
        </div>
    );
}

export default Post;
```

Post.module.css

```
.postContainer {
    width: 100%;
}

.post{
    display: flex;
    flex-direction: column;
    margin: 40px 10px 0 10px;
    border-radius: 10px;
}
```

**Comment by jamesmosier** [comment]
Hi @aaronhawkey48. I wasn't able to see a login screen, only "Hello" on your site.

Also do you have some code that you could share of your project?

**Reply by aaronhawkey48** [reply]
https://nh-companion.web.app/sign-in

And it will direct you to /feed after log in. Sorry

**Reply by jamesmosier** [reply]
Thanks! Could you share the code for the `.Card` component. Both the React component and the CSS.

**Reply by aaronhawkey48** [reply]
updated! :)

**Comment by jamesmosier** [comment, marked as answer]
You should be able to set it to `opacity: 0.95;` which in essence is the same as 95%.

It looks like setting the opacity to a percent is a [known issue](https://stackoverflow.com/a/61124900/1554860) in an upstream tool (cssnano).

https://github.com/facebook/create-react-app/issues/7980
https://github.com/cssnano/cssnano/pull/881

**Reply by aaronhawkey48** [reply]
You are awesome my guy! Thank you!

**Reply by wahyunurarizky** [reply]
thanks. i'm stuck for hour on that lol
